pkg/payload/payload.go

```diff
@@ -139,19 +162,19 @@ func LoadUpdate(dir, releaseImage, excludeIdentifier string) (*Update, error) {
 
 			raw, err := ioutil.ReadFile(p)
 			if err != nil {
-				errs = append(errs, errors.Wrapf(err, "error reading file %s", file.Name()))
+				errs = append(errs, err)
```

Comment:
Why would we want a less specific error here?

Reply:
`ReadFile` errors already include this information.